Sanity Check PR steps are failing to install podman-docker

### Bug Summary

Both "Sanity Check" PR checks are all failing with `Failed to fetch mirror+file:/etc/apt/apt-mirrors.txt/pool/universe/libp/libpod/podman-docker_3.4.4%2bds1-1ubuntu1.22.04.2_amd64.deb  404  Not Found [IP: 52.147.219.192 80]`


### Details

Full error message:
```
+ apt-get download podman-docker
Get:1 file:/etc/apt/apt-mirrors.txt Mirrorlist [142 B]
Ign:2 http://azure.archive.ubuntu.com/ubuntu jammy-updates/universe amd64 podman-docker amd64 3.4.4+ds1-1ubuntu1.22.04.2
Ign:2 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 podman-docker amd64 3.4.4+ds1-1ubuntu1.22.04.2
Ign:2 http://security.ubuntu.com/ubuntu jammy-updates/universe amd64 podman-docker amd64 3.4.4+ds1-1ubuntu1.22.04.2
Err:2 mirror+file:/etc/apt/apt-mirrors.txt jammy-updates/universe amd64 podman-docker amd64 3.4.4+ds1-1ubuntu1.22.04.2
  404  Not Found [IP: 52.147.219.192 80]
E: Failed to fetch mirror+file:/etc/apt/apt-mirrors.txt/pool/universe/libp/libpod/podman-docker_3.4.4%2bds1-1ubuntu1.22.04.2_amd64.deb  404  Not Found [IP: 52.147.219.192 80]
Error: Process completed with exit code 100.
```

Looks like an instance of https://github.com/actions/runner-images/issues/7452 to me.

--- a/.github/workflows/pr.yml
+++ b/.github/workflows/pr.yml
@@ -107,6 +107,7 @@ jobs:
         run: |
           set -x
           # Prefer using podman over docker
+          sudo apt-get update
           apt-get download podman-docker
           sudo dpkg --force-all -i podman-docker*.deb 2>/dev/null
           EXCLUDE="--exclude hack/ --exclude plugins/modules/nmcli.py"

